docs/logging.md

```diff
@@ -184,48 +176,42 @@ when the loop exits. The end loop record include the number of cycles
 actually executed.
 
 ```
-14:14:30.330 TEST_06       28 LOOP FOR J = 1  TO 4
-14:14:30.571 TEST_06       35 LOOP ENDFOR J              4
-+-----------+-------------+--+---+----- loop ------+- result -+
+10:01:16.243 LOOP   MUTE10_10    39    START   DO 
+10:01:16.631 LOOP   MUTE10_10    39    END     DO  100000
++--------------- header -------------+--- loop ---+------+
 ```
 
-The example below shows a DOW loop execution.
-
-```
-14:14:30.330 TEST_06       13 LOOP DOW LOOP START COUNT < 100 
-14:14:30.556 TEST_06       22 STMT LEAVE
-14:14:30.571 TEST_06       33 LOOP DOW LOOP END                   45	
-+-----------+-------------+--+---+---------- loop -----------+- result -+
-```
 Please note that statements like LEAVE may affect the number of cycles
 actually executed.
 
 ## Performance Channel PERF
-The performance channel measures the execution time of loops, programs and
-subroutines. 
+The performance channel measures the execution time of statements. 
 The log records are generated at the end of statements block, measuring the
-time in milliseconds.
+time in microseconds.
 
 ```
-15:09:46.910 TEST_06       79 PERF ENDFOR I                       8 ms
-15:09:46.910 TEST_06       80 PERF SUBROUTINE END PRINT           9 ms
-15:09:46.910 TEST_06          PERF END TEST_06                  160 ms
-+-----------+-------------+--+---+---------- data -----------+- result -+
+15:51:55.838 PERF   MULANGTC10      7       PLIST   elapsed 8.875us
+15:51:55.840 PERF   MULANGTC10      13      EVAL    elapsed 566us
+15:51:55.840 PERF   MULANGTC10      14      EVAL    elapsed 35.084us
++---------------- header -----------------+--------+----- perf ----+
 ```
 
 ## Parsing Channel PARS
 The parsing channel measures the parsing time needed to build AST.
 The log records are generated at the end of each phase related to parsing.
 
 ```
-12:36:33    MUTEXX        182 PARS PREPROP END MUTEXX          165 ms
-12:36:33    MUTEXX        182 PARS RPGLOAD END MUTEXX          127 ms
-12:36:33    MUTEXX            PARS LEXER END MUTEXX            236 ms
-12:36:34    MUTEXX            PARS PARSER END MUTEXX           674 ms
-12:36:41    MUTEXX            PARS RCONTEXT END MUTEXX        7191 ms
-12:36:41    MUTEXX            PARS CHKPTREE END MUTEXX          17 ms
-12:36:42    MUTEXX            PARS AST END MUTEXX              361 ms
-+-----------+-------------+--+---+---------- data ------------+- result -+
+09:55:51.994PARS  MULANGTC10            START   LEXER
```

Comment:
Parsing channel was introduced to measure the performance issues related parse tree creation (ms spent), now I have no longer this  information.